app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapFragment.java

```diff
@@ -203,6 +174,7 @@ public static ExploreMapFragment newInstance() {
     @Override
     public void onCreate(@Nullable final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        binding = FragmentExploreMapBinding.inflate(getLayoutInflater());
```

Comment:
This line could move to `onCreateView()` and delete `onCreate()` if it's doing nothing useful.